### PR TITLE
Addition to package.json

### PR DESCRIPTION
**Pull request by ev1stensberg**
Added dev engines and version types

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,8 +11,9 @@
     "lib": "evernote-sdk-js",
     "example": "sample"
   },
-  "engines": {
-    "node": "*"
+  "devEngines": {
```
Comment (akhaku) [inline package.json]: It doesn't look like `devEngines` is actually a thing - `engines` is still the standard.

Comment (ev1stensberg) [inline package.json]: `devEngines` is used by FB

**Comment by akhaku** [comment]
Thanks for your PR. I don't know if specifying the engines is necessary, since we don't currently have a version requirement. This makes it a tiny bit harder to maintain - we'd have to add versions to the engines as they become available.

**Comment by ev1stensberg** [comment]
If you want, I could help out with that :smile: Also, I prefer otherwise ( cause you'll have more end control ) , but that's me!

**Pull request closed without merging**